src/main/resources/db/changelog/v.0.5/db.changelog-user-interview-time.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
+  - changeSet:
```

Comment:
Do you use this enum somewhere? If no, remove please.

Reply:
Done